global.json

```
@@ -1,6 +1,7 @@
 {
   "sdk": {
-    "version": "8.0.101",
-    "rollForward": "latestFeature"
+    "version": "9.0.0",
```

```suggestion
    "version": "9.0.100-rc.2",
```
I think its typically like this, but if it works 🤷‍♂️